tests/integration/components/render-content-test.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@ember/component';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { find } from 'ember-native-dom-helpers';
+
+module('Integration | Component | render-content', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    // Register a simple component to test `render-content` with.
+    this.owner.register('component:my-component', Component.extend({
+      classNames: ['my-test-component'],
+      layout: hbs`{{text}}`,
+    }));
+  });
+
+  test('it renders simple content correctly', async function(assert) {
+    await render(hbs`
+      <div id="render-content-test">
+        {{render-content "blah"}}
+      </div>
+    `);
+
+    assert.equal(find('#render-content-test').textContent.trim(), 'blah');
```

Comment:
Only nitpicky suggestion would be to use `const`s for the text values that you're setting and checking, but this is a small and simple test so not a big deal.